src/utils/format.ts

```diff
@@ -200,3 +200,10 @@ export function formatPriceWithFloor(price: BigNumber): string {
   const displayPrice = amountToPrecisionDown(price, DEFAULT_DECIMALS).toString(10)
   return price.gt(LOW_PRICE_FLOOR) ? displayPrice : '< ' + LOW_PRICE_FLOOR.toString(10)
 }
+
+export function capitalize(sentence: string): string {
+  return sentence
+    .split(' ')
+    .map((word) => word[0].toUpperCase() + word.slice(1).toLowerCase())
```

Comment:
I must be reading it wrong, cause reading this code I would expect to be Sell Order, not "Sell order"

what am i missing?
![image](https://user-images.githubusercontent.com/2352112/108511430-30073c80-72c0-11eb-98bf-c74fb63068e7.png)

Reply:
I'm capitalizing only the `kind`, not the whole sentence. As per design.

Reply:
Yes, I agree, I was seeing it correct in the UI. My comment was more like, that by only looking at the code, I was thinking u were capitalizing all (which I think it would be wrong), but obviously I read bad the code cause it's all good, so u can forget this comment ever existed :)